STATUS: INLINE SUCCESSFUL
MODIFIED: net.sf.marineapi.nmea.parser.SentenceFactoryTest.testRegisterParserWithAlternativeBeginChar
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/marine-api/src/test/java/net/sf/marineapi/nmea/parser/SentenceFactoryTest.java
CLEANER: net.sf.marineapi.ais.event.TestGroup100Case0.test280
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/marine-api/src/test/java/net/sf/marineapi/ais/event/TestGroup100Case0.java
POLLUTER: net.sf.marineapi.nmea.parser.SentenceFactoryTest.testRegisterParserWithAlternativeBeginChar
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/marine-api/src/test/java/net/sf/marineapi/nmea/parser/SentenceFactoryTest.java
ORIGINAL CLEANER SIZE: 17
NEW CLEANER SIZE: 2
ELAPSED TIME: 149.6619999408722
==========================
@@ -235,0 +235,4 @@
+ {
+     net.sf.marineapi.nmea.parser.SentenceFactory sentenceFactory0 = net.sf.marineapi.nmea.parser.SentenceFactory.getInstance();
+     sentenceFactory0.reset();
+ }
